scripts/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "doc-scripts",
+  "version": "0.0.0",
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "vitest run",
+    "type-check": "tsc --noEmit",
+    "repl": "vite-node",
```

Comment:
with bun you dont need `vite-node` (here and in generate). You can just run use `bun run src/tables-metadata.ts` and it will run.

+ you can remove vite, vite-node, and vitest from the deps